### PR TITLE
Enable git-like did-you-mean feature

### PR DESCRIPTION
**Pull request by hiiwave**
As watson is a command-intensive application, I think it would be a plus if it comes with did-you-mean feature powered by [click-didyoumean](https://github.com/click-contrib/click-didyoumean), e.g.
```
$ watson strat
Usage: watson [OPTIONS] COMMAND [ARGS]...
Try "watson --help" for help.
Error: No such command "strat".
Did you mean one of these?
start
status
restart
```

What do you think about this?

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 arrow!=0.11,!=0.12.0
 click
 requests
+click-didyoumean
```
Comment (davidag) [inline requirements.txt]: I would put both click dependencies togeter.

Comment (hiiwave) [reply requirements.txt]: I'm neutral to the style here and let's see if maintainers have any preference.

Comment (jmaupetit) [reply requirements.txt]: I like when things are alphabetically sorted :wink:

**Comment by jmaupetit** [comment]
I :heart: it!

**Pull request merged**